independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/runner/Timing.java

```diff
@@ -97,15 +97,19 @@ public static void printStartupTime(String name, String version, String quarkusV
         //Use a BigDecimal so we can render in seconds with 3 digits precision, as requested:
         final BigDecimal secondsRepresentation = convertToBigDecimalSeconds(bootTimeNanoSeconds);
         String safeAppName = (name == null || name.trim().isEmpty()) ? UNSET_VALUE : name;
-        String safeAppVersion = (version == null || version.trim().isEmpty()) ? UNSET_VALUE : version;
+        String safeAppVersion = (version == null || version.trim().isEmpty() || version == "null") ? UNSET_VALUE : version;
```

Comment:
```suggestion
        String safeAppVersion = (version == null || version.trim().isEmpty() || version.equals("null")) ? UNSET_VALUE : version;
```

Reply:
Outch, what the hell...

Reply:
Funnily enough, I suppose the string was interned as it was working.

I think I'm too used to write `thing == null` without quotes :).

Thanks for catching it.